.github/dependabot.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
+updates:
+  # Enable version updates for npm
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 10
+    reviewers:
+      - "o1-labs/o1js-maintainers"
```

Comment:
Who is this?  The teams I know about are:

<img width="421" height="163" alt="image" src="https://github.com/user-attachments/assets/7db8cc1d-122c-4748-8100-7a7ef1de9b47" />

Reply:
See [5a325dd](https://github.com/o1-labs/o1js/pull/2290/commits/5a325ddb82d065d9a7ab294af5478bf4d1cd4294)